docs/source/tutorials/index.rst

```diff
@@ -34,12 +34,19 @@ your datasets and turn your good models into *great models*.
 .. Add tutorial cards below
 
 .. customcarditem::
-    :header: Evaluating object detections
+    :header: Evaluating detections on COCO
```

Comment:
I'd be careful about changing this. This tutorial is going to be a simpler and more generalizable tutorial on performing evaluation. It is backed by pycocotools, yes, but it will work on most any detection dataset. The Open Images tutorial is basically just for evaluating Open Images

Reply:
I see your point; changes reverted